pkg/controllers/vdb/podfacts.go

```diff
@@ -995,14 +999,22 @@ func (p *PodFacts) countRunningAndInstalled() int {
 	})
 }
 
-// countInstalledAndNotRestartable returns number of installed pods that aren't yet restartable
-func (p *PodFacts) countInstalledAndNotRestartable() int {
+// countNonRunningPodsNeededByRestart returns number of pods that aren't yet
+// running but the restart reconciler needs to handle them.
+func (p *PodFacts) countNonRunningPodsNeededByRestart(vclusterOps bool) int {
```

Comment:
I think this function name is not very accurate as we are not only counting non running pods

Reply:
Good point. Maybe countNotRestartablePods()?